app.go

```diff
@@ -73,19 +82,23 @@ type EventListener interface {
 }
 
 func NewApp(bcAPI *eos.API, brokerClient EventListener, eventMessages chan *broker.EventMessage,
-	offsetHandler io.Writer,
+	offsetHandler utils.FileStorage,
 	cfg *AppConfig) *App {
 	return &App{bcAPI: bcAPI, BrokerClient: brokerClient, OffsetHandler: offsetHandler,
-		EventMessages: eventMessages, AppConfig: cfg}
+		EventMessages: eventMessages, AppConfig: cfg, Metrics: GetMetrics()}
 }
 
 func (app *App) processEvent(event *broker.Event) *string {
 	log.Debug().Msgf("Processing event %+v", event)
+	start := time.Now()
+	defer func() {
+		elapsed := time.Since(start)
+		app.Metrics.SigniDiceProcessingTimeMs.Observe(elapsed.Seconds() * 1000)
```

Comment:
why not `Milliseconds()`  ?

Reply:
cause it returns an Int
but Seconds() returns a float